src/components/UserProfile.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { Container } from "reactstrap";
+
+const UserProfile = () => {
+  return (
+    <Container>
+      <div>
+        {/* Need to display user's username in Welcome*/}
+        <h1>Welcome, User!</h1>
+      </div>
+      <div>
```

Comment:
Let's add in some sort of `map` here that utilizes a Redux action to pull videos for the user